lib/ansible/modules/cloud/vmware/vmware_tag_manager.py

```diff
@@ -24,6 +24,7 @@
 version_added: 2.8
 author:
 - Abhijeet Kasurde (@Akasurde)
+- Frederic Van Reet (@GBrawl)
```

Comment:
Do you want to add to the copyright statements as well (see top)?  I usually add to the copyright and don't add myself as an author to the module unless I contributed a significant set of changes or originated the module.

Reply:
Saw this after this got merged, I'll remember for next time